cmds/ueficat/ueficat.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// ueficat will cat a set of files from a firmware volume.
+// The files are specified with a regular expression that is
+// matched against a file GUID.
+//
+// Synopsis:
+//     ueficat <romimage> pat [pat ...]
```

Comment:
Would it make sense to make this part of the `utk` tool?

        utk <romimage> cat <pattern>

Reply:
Yes but we can also leave it as a separate command

Reply:
I like the idea of showing how to write simple commands with this package, I'd like to keep this separate.